### ISSUE TITLE
Homebrew formula

### ISSUE THREAD

**Issue by 0xflotus** [hakluke/hakrevdns#15]
Is it possible to create a Homebrew formula to make installation on Mac easier?

**Comment by iocozmo** [comment]
Yes I would like this as well. Is there any way to install this on Mac?

**Comment by alexanderadam** [comment]
[Linux users will profit from this as well](https://docs.brew.sh/Homebrew-on-Linux)
